src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,67 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
```

Comment:
The `getUser` function is being imported from '../../api', but it is not clear if this function handles errors. Consider adding error handling to manage cases where the user data cannot be fetched.